tedana/decomposition/ica.py

```diff
@@ -50,6 +53,51 @@ def tedica(data, n_components, fixed_seed, maxit=500, maxrestart=10):
         "decompose the dimensionally reduced dataset."
     )
 
+    if ica_method=='robustica':
+        mmix, Iq = r_ica(data, n_components, n_robust_runs, maxit)
+        fixed_seed=-99999
+    elif ica_method=='fastica':
+        mmix, fixed_seed=f_ica(data, n_components, fixed_seed, maxit=500, maxrestart=10)
```

Comment:
I see the default values for maximum iterations and maximum restarts appearing at two different locations in the code. This is potentially facilitating a downstream error, wherein someone wants to change these default values, but they only modify the value at one of the two locations where it appears. In C / C++ I'd define one of these as a macro / `constexpr`; with Python I would think they would be best as private global consts eg. "`_DEFAULT_MAXIT`"? I've started doing this a little in Python but don't actually know for sure if it's the accepted best approach. Indeed maybe you could define a const global `dict` containing the defaults?

Reply:
Correct. Default values have been set at two or sometimes three different locations. I have set a config.py file that contains the default values. I talked to Dan about it and he suggested that I do this for parts of the code that I am working on and they will decide how to proceed sometime later. So, let’s leave it to them.